pyproject.toml

```diff
@@ -103,7 +103,8 @@ test-command = "pytest {package}/tests"
 #    PyPy
 #    musllinux when python < 3.9
 #    musllinux in aarch64
-skip = ["pp*", "cp38-musllinux*", "*-musllinux_aarch64"]
+#    python == 3.12 (until numpy supports python 3.12)
+skip = ["pp*", "cp38-musllinux*", "*-musllinux_aarch64", "cp312-*"]
```

Comment:
You don't need this. `cibuildwheel` looks at your main `python_requires` to determine which Python version to build.

Reply:
oh i was thinking that but then i wanted to wait until CI would tell me otherwise 😬 will remove